recipes/geotexxx/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "geotexxx" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geotexxx-{{ version }}.tar.gz
+  sha256: 0ca0c47e8760e8a2d0ebeb08956aa4674daa3a0ee2870e361c7ecbba5fa70538
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - requests
+    - importlib-metadata  # [py<38]
+
+test:
+  imports:
+    - geotexxx
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Package om geotechnisch grondonderzoek te verwerken
+  license: MPL-2.0
+  license_file: LICENSE.txt
+  home: https://pypi.org/project/geotexxx/
+
+extra:
+  recipe-maintainers:
+    - Thomasvanderlinden
```

Since you're contributing the recipe, you should list your username here. You're welcome to include the upstream maintainer as well if you like, but we'll need to make sure they're ok with being listed.
```suggestion
    - ic144
```